src/Intents/INPerson.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+using System.Runtime.Versioning;
+
+#nullable enable
+
+namespace Intents
+{
+
+#if !TVOS
+	public partial class INPerson
+	{
+
+#if !NET
+	[Introduced (PlatformName.iOS, 15,0)]
+	[Introduced (PlatformName.MacOSX, 12,0)]
+	[Introduced (PlatformName.WatchOS, 8,0)]
+#else
+	[SupportedOSPlatform ("ios15.0")]
+	[SupportedOSPlatform ("macos12.0")]
+#endif //!NET
+        public enum INPersonType {
```

Comment:
The indentation is off here, and it looks like the indentation is spaces too (when it should be tabs).

Reply:
ahh thanks for catching!